design/ui/cards/card/card.tsx

```diff
@@ -9,10 +9,10 @@ export type CardProps = {
   children?: ReactNode;
 } & React.HTMLAttributes<HTMLDivElement>;
 
-export function Card({ children, className, ...rest }: CardProps) {
+export const Card = forwardRef<HTMLDivElement, CardProps>((props, ref) => {
   return (
-    <div className={classNames(styles.card, className)} {...rest}>
-      {children}
+    <div ref={ref} className={classNames(styles.card, props.className)} {...props}>
+      {props.children}
```

Comment:
children are included in props, so you can skip them.

```suggestion
    <div ref={ref} className={classNames(styles.card, props.className)} {...props}/>
```